### ISSUE TITLE
Messaging “susty” vs. “sustainable”

### ISSUE THREAD

**Issue by j9t** [w3c/sustyweb#7]
An observation, if instead of “susty,” the effort used the word “sustainable” (which “susty” seems to be a minimization of), the effort would seem more credible, and send a stronger signal.

Given the importance of the topic, it seems to be an unnecessary risk to surprise or confuse people, instead of being clear about the topic.

(Apart from that, using “sustainable” could also improve discoverability.)

**Comment by lmastalerz** [comment]
Thanks for the suggestion @j9t ! We've got so used to our internal "lingo" over time that we stopped noticing that it could be less than obvious for external consumers. I'm very much in favour of updating.

**Comment by AlexDawsonUK** [comment]
Agreed. I'll mark this as a roadmap item to ensure that going forward we always externally refer to sustainability and only use the term sustyweb if required in an internal W3C context (or as part of a URL).

**Comment by airbr** [comment]
Im just a random person but I saw this and agree that sustainable versus susty makes sense.

**Comment by borisschapira** [comment]
If we really need to shorten this, why not "s12y". It makes sense regarding other W3C usages and has already been used in W3C-issued materials https://www.w3.org/2021/10/19-sustainability-minutes.html

**Comment by AlexDawsonUK** [comment]
I think it's important to establish here (as this conversation seems to be getting popular) that we haven't knowingly attempt to shorten the term sustainability that I'm aware of (at least not publicly - outside of the CG or when mentioning the URL to W3C circles).
When the community group group was first established there was already a Sustainability CG in existence working on similar but non-conflicting materials, and as you cannot have two conflicting names the "sustyweb" title was chosen at the time to avoid conflict (and for URL purposes).
Our community group itself is called **Sustainable Web Design**, and neither the specification, or any supplementary documents at any point use an abbreviated version (such as sustyweb) reduction of the term sustainability. The term only appears in use within the direct URL to our CG and GitHub (as required) to ensure the anchors maintain active.

**Comment by AlexDawsonUK** [comment]
This was discussed at our monthly meeting (27/09/23) so members are now aware of the issue.
I'll mark this issue as resolved, however if it re-occurs in the future it can be re-opened for discussion.

**Comment by j9t** [comment]
If I may (appreciate you having been open to the first feedback)—upon reviewing, the URL slug (“sustyweb”) and repo URLs (“sustyweb”) still draw attention and undercut the messaging.

GitHub redirects repositories once renamed (so that should not inconvenience anyone), but I’m not sure how easy it is to redirect URLs on github.io. Was that a reason not to rename more mentions of “susty”?

**Comment by AlexDawsonUK** [comment]
There's other logistical factors at work aside from the GitHub repo.

Our W3C CG title is "Sustainable Web Design" (Short URL [SustyWeb](https://www.w3.org/community/sustyweb/)). Aside from the factor that the ability to change a CG's name and URL once created (this was done in 2013) isn't within the capability of it's chairs (I'm not even sure if it's possible but we'd have to examine this with W3C members further), when we chose to begin crafting a specification a Sustainability CG was already in existence (they are a different group from us examining a different cohort of issues so not to be confused with our work). So we would first need to get senior W3C members to change our CG name (once a new name has been agreed upon), then look into setting up some kind of redirect (if such a scenario is possible for their CG system), and then once moved to a new location we could look to renaming assets such as within the GitHub (etc).

In short, it's a huge undertaking involving a system we have no control over, redirecting a website & a wiki & a github repo & updating SpecRef, and thats just the things off the top off my head.

What we had planned during our meeting is that in the short term, we would simply discontinue any mention through communication of the sustyweb term (aside from using the URL as a necessary reference). For a long-term resolution (as remember we are still in early draft stage), as we plan to eventually officialize our work and look to potentially join our spec under the banner of a sustainability working group. We would resolve this issue by default as the specification would move with our materials to a permanent home under the long-term care of the W3C (where we would continue to work on it).

**Comment by j9t** [comment]
Thank you—appreciate you sharing this context and detail!

**Comment by AlexDawsonUK** [comment]
**Note:** Self-reference that as this remains an issue which will be present until / if we become a Working Group, I'll mark this as not planned (as yet) so that it will remind me to revisit the subject at the right time to ensure the issue is addressed.
